.github/ISSUE_TEMPLATE/request-for-amplification-template.yml

```diff
@@ -1,28 +1,54 @@
 name: Request for amplification
-description: Request distribution of material by the Marketing Team to various marketing channels
+description: Request amplification by the Marketing Team to various marketing channels
 title: "[AMPLIFY]: "
-labels: ["amplify"]
+labels: ["amplification-request"]
 assignees:
+  - sereedmedia
+  - robinwpdeveloper
   - eidolonnight
 body:
   - type: markdown
     attributes:
       value: |
-        Need help spreading the news? Let us know what you would like amplified across the WordPress ecosystem.
-  - type: input
-    id: date
+        Need help spreading the news? The Marketing team can help amplify MakeWP content across the WordPress ecosystem.
+  - type: dropdown
+    id: request-type
     attributes:
-      label: Post date
-      description: What is your preferred day and time to post? Please note that posts to social platforms are scheduled at least 1-week in advance. For urgent requests, please consult the [#marketing Slack](https://wordpress.slack.com/archives/C0GKJ7TFA).
-      placeholder: ex. 24 May 2022
+      label: What is the goal of this amplification request? (select all that apply)
+      multiple: true
+      options:
+        - Inform community/make an announcement (no action necessary)
+        - Solicit feedback
+        - Recruit volunteers
+        - Other
     validations:
-      required: false
+      required: true
   - type: dropdown
-    id: channels
+    id: audience
     attributes:
-      label: Where would you like this shared?
+      label: What is the intended audience(s) for this content? (select all that apply)
       multiple: true
       options:
+        - Any/All/Don't know
+        - General WordPress community
+        - MakeWP community
+        - WordPress users (existing)
+        - WordPress users (new/potential)
+        - Builders/implementers
+        - Developers
+        - Designers
+        - Plugin community
+        - Theme community
+        - Agencies/Enterprise
```

Comment:
May be `Agencies/Enterprises`

Reply:
"Enterprise" in terms of enterprise-level business using WordPress. But I think I should separate those anyway.